twarep/electrifly
# Labelled remainder of flights, 68 at this time

## Checklist

## What I Did
**ml-label.ipynb**: Added code at the bottom to label any incoming flights, in this case I hardcoded to exclude the 6 flights already in the database. This code does not generalize to new flights, because it doesn't consider flights that are already labelled.

**flight_querying.py**: Added some queries to simplify the ml-label code as it relates to getting and pushing data into the database.


## How I Did it?
What packages did I install to accomplish this goal? What libraries did I need that the team might not know?
Write anything in this section that is technically related that everyone on the team might not know or is not
trivial.

## How to Test?
Manual testing? Automated testing? E2E testing? Smoke tests?

Check the db for which flights are in the flight_activities table, can run different queries to analyze the activities.

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
